### DISCUSSION TITLE
Is it compatible with all browsers?

### DISCUSSION THREAD

**Discussion by kriptoescobar007** [teaxyz/white-paper#167] [Q&A]
Even though I have logged in and created my user profile, it gets stuck on the lets do this screen.

Is it compatible with all browsers?

![2](https://github.com/teaxyz/white-paper/assets/134381613/f716db4c-fb17-45b0-9a98-1082391a7682)

**Comment by ronliang521** [comment]
Don't worry, this is normal. Many testers have encountered this problem. I believe that as long as we wait quietly, the problem will be solved. My guess is that there are too many testers.

**Reply by kriptoescobar007** [reply]
It seems more like a browser incompatibility or plugin error to me. But of course I keep waiting.

**Reply by ronliang521** [reply]
No, it's not. This is not a browser compatibility issue. I was able to log in at first, but then I encountered the same problem as you when I tried to log in again. It's probably because there are too many people

**Reply by kriptoescobar007** [reply]
ok thanks for info.

**Reply by ronliang521** [reply]
You are welcome

**Comment by zeroXserdar** [comment]
Can you try the following:

**1.** Right click on the page and select 'Inspect'
![image](https://github.com/teaxyz/white-paper/assets/19645974/cb20701e-e474-4cfa-bf6e-578124314a41)

**2.** Select 'Application' from the top bar, select 'Storage' from the left side bar, check 'include third-party cookies', then eventually click 'Clear site data'.
![image](https://github.com/teaxyz/white-paper/assets/19645974/fa3f196f-0f0e-446f-8e37-5c9b1bacd84a)

Hopefully it solves your issue. Good luck!

Discord: iamserdar

**Reply by kriptoescobar007** [reply]
The problem was the density, I logged in today and there was no problem. But thank you for the good information.
